Revise the positionConfig contents (#650)
DevExpress/devextreme-documentation: api-reference/50 Common/Object Structures/positionConfig/my/my.md

---
---
##### shortDescription

> Specifies a side or corner of the overlay element that should be aligned with the [target element](/Documentation/ApiReference/Common/Object_Structures/positionConfig/#of).

Reviewer: ```suggestion
Specifies the overlay element's side or corner to align with a [target element](/Documentation/ApiReference/Common/Object_Structures/positionConfig/#of).
```

---

> To set this option, use an object with the **x** and **y** fields or one of the string shortcuts listed in the accepted values.

Reviewer: Can we be more specific than "use an object with the **x** and **y** fields"
(See my similar comments above)

"or one of the string shortcuts listed in the accepted values"
Change to "or a string shortcut from the accepted values list"
